tests/distribution_test.go

```diff
@@ -107,33 +106,73 @@ func createUpdate(t *testing.T) {
 	bundleName := initLocalDistributionTest(t, "client-test-bundle-2")
 	defer deleteLocalBundle(t, bundleName, true)
 
-	// Create unsigned release bundle
+	// Create release bundle params
 	createBundleParams := services.NewCreateReleaseBundleParams(bundleName, bundleVersion)
 	createBundleParams.Description = "Release bundle description 1"
 	createBundleParams.ReleaseNotes = "Release notes 1"
 	createBundleParams.SpecFiles = []*utils.ArtifactoryCommonParams{{Pattern: RtTargetRepo + "b.in"}}
-	err := testsBundleCreateService.CreateReleaseBundle(createBundleParams)
-	assert.NoError(t, err)
+
+	// Test DryRun first
+	err := createDryRun(createBundleParams)
```

Comment:
Since this is a dry-run, we can validate that the release bundle was not created, in addition to the err check we have now.
The same goes for the ```updateDryRun``` action - we can check that the bundle is not modified after the operation.